### DISCUSSION TITLE
Cyberduck ask me to buy a license every time I restart my mac

### DISCUSSION THREAD

**Discussion by Jacob-Palme** [iterate-ch/cyberduck#16438] [Q&A]
I have got a license for Cyberduck a long time ago. But every time I restart my Mac computer, it asks me to get a license even though I already have one.

**Comment by tatze96** [comment]
Please verify that the license file is present within the [application support folder](https://docs.cyberduck.io/cyberduck/support/#application-support-folder) and that it has the correct file extension (*.cyberducklicense*) once you get the message again. If the registration key file isn't present, add it to the application support folder.

If you still get the message after a restart afterward, check for scripts or third-party applications that reset the system to a certain state during system shutdown. If you have one in use, ensure the [application support folder](https://docs.cyberduck.io/cyberduck/support/#application-support-folder) is excluded so that Cyberduck can work correctly and the license file remains present.
